src/modules/page/admin/main.php

```diff
@@ -24,69 +23,59 @@
     nv_redirect_location(NV_BASE_ADMINURL . 'index.php?' . NV_LANG_VARIABLE . '=' . NV_LANG_DATA . '&' . NV_NAME_VARIABLE . '=' . $module_name . '&' . NV_OP_VARIABLE . '=content');
 }
 
-$array_status = [
-    $nv_Lang->getModule('inactive'),
-    $nv_Lang->getModule('active')
-];
-
-$xtpl = new XTemplate('main.tpl', NV_ROOTDIR . '/themes/' . $global_config['module_theme'] . '/modules/' . $module_file);
-$xtpl->assign('LANG', \NukeViet\Core\Language::$lang_module);
-$xtpl->assign('GLANG', \NukeViet\Core\Language::$lang_global);
-
-$i = 0;
-foreach ($_rows as $row) {
-    $row['url_view'] = NV_BASE_SITEURL . 'index.php?' . NV_LANG_VARIABLE . '=' . NV_LANG_DATA . '&amp;' . NV_NAME_VARIABLE . '=' . $module_name . '&amp;' . NV_OP_VARIABLE . '=' . $row['alias'] . $global_config['rewrite_exturl'];
-    $row['url_edit'] = NV_BASE_ADMINURL . 'index.php?' . NV_LANG_VARIABLE . '=' . NV_LANG_DATA . '&amp;' . NV_NAME_VARIABLE . '=' . $module_name . '&amp;' . NV_OP_VARIABLE . '=content&amp;id=' . $row['id'];
-    $row['checkss'] = md5($row['id'] . NV_CHECK_SESSION);
-    $row['hitstotal'] = nv_number_format($row['hitstotal']);
-    for ($i = 1; $i <= $num; ++$i) {
-        $xtpl->assign('WEIGHT', [
-            'w' => $i,
-            'selected' => ($i == $row['weight']) ? ' selected="selected"' : ''
-        ]);
-
-        $xtpl->parse('main.row.weight');
-    }
-
-    foreach ($array_status as $key => $val) {
-        $xtpl->assign('STATUS', [
-            'key' => $key,
-            'val' => $val,
-            'selected' => ($key == $row['status']) ? ' selected="selected"' : ''
-        ]);
+$_query = $db->query("SELECT config_value  FROM " . NV_PREFIXLANG . "_" . $module_name . "_config WHERE config_name='copy_page'");
+while ($row = $_query->fetch()) {
+    $config_page = $row;
+}
```

Comment:
@nambmtk1993 tại file `admin.functions.php` nó đã lấy sẵn toàn bộ config vào biến `$page_config`, em chỉ việc dùng biến đó chứ không lấy lại vào CSDL thêm nữa cũng thừa